docs/source/conf.py

```diff
@@ -12,18 +12,19 @@
 #
 import os
 import sys
-sys.path.insert(0, os.path.abspath('../..'))
-print('docs path', sys.path[0])
-os.environ['POLYGON_API_KEY'] = 'POLYGON_API_KEY'
+
+sys.path.insert(0, os.path.abspath("../.."))
+print("docs path", sys.path[0])
+os.environ["POLYGON_API_KEY"] = "POLYGON_API_KEY"
 
 # -- Project information -----------------------------------------------------
 
-project = 'polygon-api-client'
-copyright = '2022, Polygon.io'
-author = 'Polygon.io'
+project = "polygon-api-client"
+copyright = "2022, Polygon.io"
+author = "Polygon.io"
```

Comment:
Can we not make any changes to this file. It doesn't seem related to the proposed changes.

Reply:
this is from running poetry run black, its auto formatting.